troop/urls.py

```diff
@@ -0,0 +1,29 @@
+from django.urls import include, path
+from . import views
+
+app_name = "troop"
+urlpatterns = [
+    path(
+        "<int:troop>/",
+        include(
+            [
+                path("", views.IndexView.as_view(), name="index"),
+                path(
+                    "participants/",
+                    views.IndexParticipantView.as_view(),
+                    name="participant.index",
+                ),
+                path(
+                    "participants/<int:pk>/",
```

Comment:
I think we should use either singular or plural in the URLs: `/troop/<int:troop>/participants/<int:pk>/`

Reply:
I say it more like:
- this is the "troop" app (for management inside a given troop)
- inside it, we specify the troop we want to manage and then it is more "REST-like" (were plural are more usual)

For `troop`, the user can't really do anything on it (he can't create or delete a troop), whereas for `participants` a complete CRUD is available.

However I am not really tied to this, if you really prefer, we can switch to singular everywhere (which Django-Admin is using).

Reply:
Hm. Yeah, I just had the Django dashboard in mind. I think it is a bit weird to have something like `participants/3/edit`. But I'm not tied to this either, so I'd say it's up to you if you want to change it.

Reply:
:+1: let's go singular all the way!